src/shared/components/CodeSnippet.tsx

```diff
@@ -109,9 +114,19 @@ const CodeSnippet: FC<Props> = ({
         className="code-snippet--scroll"
       >
         <div className="code-snippet--text">
-          <pre>
-            <code>{_text}</code>
-          </pre>
+          {language === null ? (
+            <pre>
+              <code>{_text}</code>
+            </pre>
+          ) : (
+            <SyntaxHighlighter
+              language={language}
+              style={syntaxTheme}
```

Comment:
If the component has a `language` specified, use it, else keep the default `purple` font.